plugins/catalog-backend-module-aws/api-report.md

```diff
@@ -30,6 +33,14 @@ export class AwsOrganizationCloudAccountProcessor implements CatalogProcessor {
   ): Promise<boolean>;
 }
 
+// @public (undocumented)
+export type AwsS3Config = {
```

Comment:
Just realized that this is not needed anymore to be exported since the factory's fromConfig was integrated and the previous create method is gone. The constructor is private, too.

Reply:
removed